ui/webui/test/check-basic

```diff
@@ -126,20 +128,48 @@ class TestQuit(anacondalib.VirtInstallMachineCase):
         m = self.machine
         i = Installer(b, m)
         l = Language(b, m)
+        s = Storage(b, m)
+
+        # BIOS boot /boot on ext4 / on xfs /home on btrfs
+        disk = "/dev/vda"
+        dev = "vda"
+        s.partition_disk(disk, [("1MiB", "biosboot"), ("1GB", "ext4"), ("10GiB", "xfs"), ("", "ext4")])
+
+        # Two partitions on the same disk with the same UUID raise exception in
+        # Blivet
+        s.set_partition_uuid(disk, 3, "66f47a35-b00e-4341-8f31-f4855fac24a2")
+        s.set_partition_uuid(disk, 4, "66f47a35-b00e-4341-8f31-f4855fac24a2")
+
+        s.udevadm_settle()
 
         i.open()
+        i.next()
 
-        l.locale_option_visible('de_DE')
         b.wait_not_present("#critical-error-modal")
 
-        # Mess up the anaconda bus address and make sure we get a critical error
-        m.execute("cp /run/anaconda/bus.address /tmp/; echo trash > /run/anaconda/bus.address")
-        b.wait_visible("#critical-error-modal")
+        s.rescan_disks()
 
-        # Errors should persist across page reloads (same as having the error prior the first page load)
-        b.reload()
         b.wait_visible("#critical-error-modal")
 
+        # Test review of logs
+
+        logfile = "/tmp/webui.log"
+
+        self.assertFalse(self.file_exists(logfile))
+
+        # Click report issue: save reviewed log and open BZ URL
+        b.click("#critical-error-modal .pf-c-button.pf-m-primary")
+
+        with b.wait_timeout(50):
+            b.wait(lambda: self.file_exists(logfile))
+
+        # Reviewed log contains more than 100 lines
+        wc_output = m.execute(f"wc -l {logfile}")
+        lines = int(wc_output.partition(" ")[0])
+        self.assertTrue(lines > 100)
+
+        sleep(3)
```

Comment:
Pleae do above: testlib.wait(lambda: int(m.execute(f"wc -l {logfile} | cut -d" " -f1")).strip()) > 100)

Reply:
did not test the exact command but should be something like this

Reply:
Used